lib/pipeline.js

```diff
@@ -191,6 +192,54 @@ class PipelineModel extends BaseModel {
     }
 
     /**
+     * Fetch all the active jobs that belong to this pipeline
+     * Sorted by workflow order, followed by PR jobs
+     * @return {[type]} [description]
+     */
+    getActiveJobs() {
+        const listConfig = {
+            params: {
+                pipelineId: this.id,
+                archived: false
+            },
+            paginate: {
+                count: PAGINATE_COUNT, // This limit is set by the matrix restriction
```

Comment:
This is not correct, there is no 25 limit when you are asking for jobs + PRs.  You need to either ask for 100+ or iterate on this.